backend/src/routes/payments.js

```diff
@@ -0,0 +1,58 @@
+const express = require('express');
+const services = require('../services');
+const router = express.Router();
+const db = require('../db');
+const auth = require('../auth');
+const utils = require('../utils');
+
+const getPaymentRedirectUrl = async (req, res) => {
+    try {
+        const paymentObj = req.body;
+        const validationErrors = utils.payments.validateAmount(
+            paymentObj.amount
+        );
+        if (validationErrors) {
+            res.status(422).json(validationErrors);
+        } else {
+            const user = req.user;
```

Comment:
I guess this line is a little obsolete as it is only used in the next line and the `req.user` is barely even longer than `user`.